package.json

```
@@ -6,6 +6,7 @@
   "private": true,
   "scripts": {
     "dev": "nuxt -p 4567",
+    "api": "json-server db.json -p 4567",
```

@miralelnahas there are about 65535 ports that you can use and you chose the exact one that the front end is running on :"D?  
```suggestion
    "api": "json-server db.json -p 4568",
```